coalib/settings/ConfigurationGathering.py

```diff
@@ -136,7 +136,7 @@ def warn_config_absent(sections, argument, log_printer):
     return False
 
 
-def load_configuration(arg_list, log_printer, arg_parser=None):
+def load_configuration(arg_list, log_printer, arg_parser=None, silent=False):
     """
     Parses the CLI args and loads the config file accordingly, taking
     default_coafile and the users .coarc into account.
```

Comment:
The message `The default coafile '.coafile' was not found.......` we get when the default coafile was not found( `load_config_file` is responsible for that. ) . so if `silent = True` then that statement is not executed under `load_config_file` . And when we are just checking for bears we don't need coafile . 

So while calling `load_configuration` from `get_filtered_bears` if we pass `silent=True` with `load_configuration` then we can use this `silent` to not warn. 

hope this helps :) . 

Can you help me with tests? :)